### PR TITLE
docs: added README.md file

### PR DESCRIPTION
**Pull request by pradipgharge**
Added the readme file for the component library.
You can browse the preview here : https://github.com/pradipgharge/Linear-UI/tree/README-file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+Toasts are intended to provide feedback to a user and self-dismiss after a short time with an option to self-dismiss. Linear UI has following type of toasts :
+- Simple Toasts
+- Toasts with buttons
+
+You can check more about the `toasts` here : https://linearui.netlify.app/pages/components/toast/toast
+
+---
+
+Thanks for reading, hope you like Linear UI.
+
```
Comment (mehulsatardekar) [inline README.md]: remove these blank spaces and you are good to merge this branch

**Review by mehulsatardekar — APPROVED** [review]

**Review by nikhiljugale007 — COMMENTED** [review]
One Suggestion, you can add link to home page of you cl in starting of readme.md
Rest looks good.

File: README.md (hunk 1 above)
Comment (pradipgharge) [reply README.md]: done

**Comment by pradipgharge** [comment]
> One Suggestion, you can add link to home page of you cl in starting of readme.md Rest looks good.

Done!

**Pull request merged**